projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -78,8 +78,12 @@ function () {
 
 		// Activate the module as the plugin is activated
 		add_action( 'admin_init', array( $this, 'activate_module_on_plugin_activation' ) );
-
-		My_Jetpack_Initializer::init();
+		add_action(
+			'plugins_loaded',
+			function () {
+				My_Jetpack_Initializer::init();
+			}
+		);
```

Comment:
What's the reasoning for moving this to a hook? Did something break otherwise?

Reply:
It could be related to the config code and the initialisation of Publicize, that happens on `plugins_loaded` but I'll let @spsiddarthan answer that.

There's a mixture of initialising in a hook or the constructor in the other products, so it should be fine either way.

Reply:
I move it to the plugins_loaded hook because I saw that in the package's README file, and assumed that was the standard. It works without this change as well.